### ISSUE TITLE
do you have a grammar description file for this script?

### ISSUE THREAD

**Issue by xiaolzha** [Cirru/cirru-script#2]

**Comment by tiye** [comment]
What kind of descriptions do you want?

By now there's only some introductions and exmaples. They may help.
http://cirru.org/
https://github.com/Cirru/cirru-parser/tree/master/cirru

**Comment by xiaolzha** [comment]
I meant a LALR or LL grammar description file that can be used to generate a parser

**Comment by tiye** [comment]
@Garriot no, parser generator are not good at paring indentations, while that's the most important part of Cirru's syntax. And CirruScript might be different with most language which does lexing and parsing before geting an AST. CirruScript is first parsed into a tree defined by Cirru project, that then that tree is converted to a AST. I may write a better guide one day. But probably I will not present a LL definition.
